package.json

```diff
@@ -194,7 +194,8 @@
 		"lint-css": "wp-scripts lint-style '**/*.scss'",
 		"lint-css:fix": "npm run lint-css -- --fix",
 		"lint-types": "tsc",
-		"lint-md": "wp-scripts lint-md",
+		"lint:md-js": "wp-scripts lint-md-js",
```

Comment:
We could bring some structure to the names of all those scripts in the main `package.json` file as an independent patch 😄

Reply:
Ticket opened: https://github.com/WordPress/gutenberg/issues/19949